### DISCUSSION TITLE
How to install?

### DISCUSSION THREAD

**Discussion by RikvdReijen** [waldobronchart/ShortcutMapper#56] [Q&A]
Dear Shortcut community,

This might be a bit of a noob question but how do I install the application? I saw some installation instructions however nothing that I could easily understand & implement. Is there a .msi or .exe installer version available somewhere (or a video showing this is installed?)

**Comment by BertranK** [comment]
Dear RikvdReijen!

If you don't need web scraping through Python and want to change only the shortcuts for a particular program,
Copy the whole thing and modify the json data in the \content\generated path and modify apps.js!
That's what I'm using!

Kim
